py/server/deephaven/table.py

```diff
@@ -694,6 +696,63 @@ def to_string(self, num_rows: int = 10, cols: Union[str, Sequence[str]] = None)
         except Exception as e:
             raise DHError(e, "table to_string failed") from e
 
+    def diff(self, table: Table, max_lines: int = 1, floating_inexact: bool = False,
```

Comment:
This is a very strange kind of diff.  Would users find the name confusing?  Would they expect "diff" of tables to return a string?  Would a different name result in less confusion?  What is a user really expecting?  Who is the intended user?  What do we expect them to use it for?